### PR TITLE
[feature] #301: render data model schema

### LINKED ISSUE #301: Proposal: render Iroha schema as Markdown
It might be useful to get `schema.json` artifact from `iroha2-dev`, `iroha2-stable` and `iroha2-lts` branches and render it in the documentation as readable Markdown. Schema types might refer to each other by anchors within a file. All schemas might be located at `/data-model-schema` guide route.

For example, we have such an entry:

```json
{
  "iroha_crypto::PublicKey": {
    "Struct": {
      "declarations": [
        {
          "name": "digest_function",
          "ty": "iroha_crypto::Algorithm"
        },
        {
          "name": "payload",
          "ty": "Vec<u8>"
        }
      ]
    }
  }
}
```

It might be rendered as:

> #### `iroha_crypto::PublicKey`
>
> **Type:** Struct
>
> **Declarations:**
> 
> | Field name | Type |
> | --: | :-- |
> | `digest_function` | [`iroha_crypto::Algorithm`](#iroha_crypto__Algorithm) |
> | `payload` | [`Vec<u8>`](#vec__u8)  |

What do you think? I see it as a relatively easy task to implement. I can even reuse some schema-related code from the JS SDK.

### PR DESCRIPTION
**Pull request by 0x009922**
Resolves #301

### CODE REVIEW

**Review by 6r1d — APPROVED** [review]
LGTM

**Comment by outoftardis** [comment]
can we render this not as a list?

image

I think what you had in the issue description looks better.

image


Otherwise looks good. We should also link to this page from the guide.

**Comment by 0x009922** [comment]
Just found that Iroha repo has schema in `docs/source/reference`, so it is easy to fetch it from GitHub, avoiding extraction through Action artifacts.

**Comment by 0x009922** [comment]
I had to update packages in order to use [dynamic routes](https://vitepress.dev/guide/routing#dynamic-routes) feature.

File: src/data-model-schema/[channel].paths.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+} satisfies Record<'lts' | 'dev' | 'stable', Schema>
+
+export default {
+  paths() {
+    const CHANNELS = ['stable', 'lts', 'dev'] as const
+
+    return CHANNELS.map((channel) => {
+      const json = MAP[channel]
+      const schema = renderSchema(json)
+      const content = `# Data Model Schema on \`iroha2-${channel}\`\n\n${schema}`
```
Comment (outoftardis) [inline src/data-model-schema/[channel].paths.ts:19]:
```suggestion
      const content = `# Data Model Schema in \`iroha2-${channel}\`\n\n${schema}`
```
?

**Review by outoftardis — APPROVED** [review]
lgtm

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**